tests/xfstests/run.pm

```diff
@@ -377,13 +377,13 @@ sub run {
         eval {
             power_action('reboot', keepconsole => is_pvm);
             reconnect_mgmt_console if is_pvm;
-            $self->wait_boot(in_grub => 1, bootloader_time => 60);
+            $self->wait_boot;
```

Comment:
I guess you need an if-else here to differentiate between spvm and other backends.

Reply:
I looked in to opensusebasetest.pm this in_grub only affect uefi thingy, and xfstests not test with uefi, so it's not important here.
For bootloader_time the default is 100, it's better than 60.
So it's not bad to keep it like this.